react-intl/index.d.ts

```diff
@@ -198,11 +156,14 @@ declare namespace ReactIntl {
 
 
     namespace FormattedPlural {
-        export interface PropsBase {
+        export interface Base {
```

Comment:
`Base` interface is used in both for `formatPlural` but `BaseProps` also contains two extra properties used in `FormattedPlural` component's props, so it's basically to make common part abstract.